Refresh.GameServer/Types/Levels/Categories/ByUserLevelCategory.cs

```diff
@@ -24,7 +24,7 @@ internal ByUserLevelCategory() : base("byUser", "by", true)
         LevelFilterSettings levelFilterSettings, GameUser? user)
     {
         // Prefer username from query, but fallback to user passed into this category if it's missing
-        string? username = context.QueryString["u"];
+        string? username = context.QueryString["username"];
         if (username != null) user = dataContext.Database.GetUserByUsername(username);
```

Comment:
IIRC its like this because `u` is used by the game, lets do `username` as a fallback.

```suggestion
        string? username = context.QueryString["u"] ?? context.QueryString["username"];
        if (username != null) user = dataContext.Database.GetUserByUsername(username);
```